### PR TITLE
collab :)

### PR DESCRIPTION
**Pull request by devjiwonchoi**

### CODE REVIEW

File: packages/create-next-app/index.ts [hunk 1]
```diff
@@ -396,7 +402,8 @@ async function run(): Promise<void> {
         program.app = Boolean(appRouter)
       }
     }
-    if (!process.argv.includes('--turbo') && !process.argv.includes('--no-turbo')) {
+
+    if (!program.turbo && !process.argv.includes('--no-turbo')) {
```
Comment (devjiwonchoi) [inline packages/create-next-app/index.ts]:
```suggestion
    if (!program.turbo) {
```

**Comment by devjiwonchoi** [comment]
@Arindam200 Hi, feel free to ask question if any doubt on changes!

**Comment by devjiwonchoi** [comment]
Or else we can squash the changes :)

**Comment by devjiwonchoi** [comment]
cc @mertcanaltin

File: packages/create-next-app/index.ts [hunk 2]
```diff
@@ -396,7 +402,8 @@ async function run(): Promise<void> {
         program.app = Boolean(appRouter)
       }
     }
-    if (!process.argv.includes('--turbo') && !process.argv.includes('--no-turbo')) {
+
+    if (!program.turbo) {
```
Comment (mertcanaltin) [inline packages/create-next-app/index.ts:406]:
```suggestion
    if (!process.argv.includes('--turbo')) {
```

Comment (devjiwonchoi) [reply packages/create-next-app/index.ts:406]: It's a program.turbo since it uses `commandjs`, where `.option` passes value as turbo there

Comment (devjiwonchoi) [reply packages/create-next-app/index.ts:406]: e.g. `--empty` to `.option` passes `program.empty` value in [Commander](https://www.npmjs.com/package/commander#common-option-types-boolean-and-value)

**Comment by mertcanaltin** [comment]
I think we need to add template type

https://github.com/vercel/next.js/pull/65927/files#diff-5204d86f816b645081eedf1a017c17df3688402c193cc16aa19b6b91a51324b9R12

**Comment by devjiwonchoi** [comment]
Since we're not adding another template, IMHO we don't need add a template type?
This Issue is mainly reading the arg `--turbo` and add it to package.json script.dev as `next dev --turbo`

Template types seem to targeting for [templates](https://github.com/vercel/next.js/tree/canary/packages/create-next-app/templates).

File: packages/create-next-app/index.ts (hunk 2 above)
Comment (mertcanaltin) [reply packages/create-next-app/index.ts:406]: I got it, thank you, this was the general use, so I suggested this

Comment (mertcanaltin) [reply packages/create-next-app/index.ts:406]: I understand, thank you very much. I suggested it because this was the general usage :+1:

**Comment by Arindam200** [comment]
> Since we're not adding another template, IMHO we don't need add a template type? This Issue is mainly reading the arg `--turbo` and add it to package.json script.dev as `next dev --turbo`

I also think so.

File: packages/create-next-app/index.ts (hunk 2 above)
Comment (devjiwonchoi) [reply packages/create-next-app/index.ts:406]: Awesome, thank you for reviewing as well 😌

**Comment by mertcanaltin** [comment]
> Since we're not adding another template, IMHO we don't need add a template type? This Issue is mainly reading the arg `--turbo` and add it to package.json script.dev as `next dev --turbo`
> 
> Template types seem to targeting for [templates](https://github.com/vercel/next.js/tree/canary/packages/create-next-app/templates).

Ah yes there is no turbo templating I guess we don't need to add it, we can do it in a separate pr  ❤️ 🚀

**Comment by Arindam200** [comment]
> > Since we're not adding another template, IMHO we don't need add a template type? This Issue is mainly reading the arg `--turbo` and add it to package.json script.dev as `next dev --turbo`
> > Template types seem to targeting for [templates](https://github.com/vercel/next.js/tree/canary/packages/create-next-app/templates).
> 
> Ah yes there is no turbo templating I guess we don't need to add it, we can do it in a separate pr ❤️ 🚀

Yes. So, we can merge this PR

**Comment by devjiwonchoi** [comment]
Cool, let's ship it!!

**Comment by Arindam200** [comment]
Merging this!

**Comment by mertcanaltin** [comment]
I agree with you, maybe we can also offer another pr :rocket:

**Pull request merged**